src/js/0-global.js

```diff
@@ -141,21 +141,24 @@ function detectOS() {
 // when using this function, pass in the name of the repo (options: releases, nightly)
 function loadJSON(repo, filename, callback) {
   var url = ('https://raw.githubusercontent.com/AdoptOpenJDK/' + repo + '/master/' + filename + '.json'); // the URL of the JSON built in the website back-end
-
   if(repo === 'adoptopenjdk.net') {
     url = (filename);
   }
-
   var xobj = new XMLHttpRequest();
-  xobj.open('GET', url, true);
+  xobj.open('GET', url, true)
   xobj.onreadystatechange = function () {
     if (xobj.readyState == 4 && xobj.status == '200') { // if the status is 'ok', run the callback function that has been passed in.
       callback(xobj.responseText);
     } else if (
       xobj.status != '200' && // if the status is NOT 'ok', remove the loading dots, and display an error:
       xobj.status != '0') { // for IE a cross domain request has status 0, we're going to execute this block fist, than the above as well.
-      loading.innerHTML = '';
-      document.getElementById('error-container').innerHTML = '<p>Error... there\'s a problem fetching the releases. Please see the <a href=\'https://github.com/AdoptOpenJDK/openjdk-releases/releases\' target=\'blank\'>releases list on GitHub</a>.</p>';
+        if (filename !== 'jck') {
+          document.getElementById('error-container').innerHTML = '<p>Error... there\'s a problem fetching the releases. Please see the <a href=\'https://github.com/AdoptOpenJDK/openjdk-releases/releases\' target=\'blank\'>releases list on GitHub</a>.</p>';
+          loading.innerHTML = '';
+        } else {
+          loading.innerHTML = '';
```

Comment:
So if we error out trying to get a JCK release we say nothing?

Reply:
the thought behind this is that not all repos will neccessarily have a jck.json file